Marlin/ultralcd.cpp

```diff
@@ -3185,7 +3185,7 @@ void kill_screen(const char* lcd_msg) {
       currentScreen = menu_edit_callback_ ## _name; \
       callbackFunc = callback; \
     } \
-    void _menu_action_setting_edit_accessor_ ## _name (const char * const pstr, _type (*pget)(), void (*pset)(_type), const _type minValue, const _type maxValue) { \
+    void _menu_action_setting_edit_accessor_ ## _name (const char * const pstr, _type (*pget)(), void (*pset), const _type minValue, const _type maxValue) { \
```

Comment:
It seems like this should remain as it is. The function signature of the setter does need to include an argument of the given type.